src/Stratis.Bitcoin.Features.PoA/PoAMiner.cs

```diff
@@ -175,6 +175,9 @@ private async Task CreateBlocksAsync()
 
                     if (chainedHeader == null)
                     {
+                        int attemptDelayMs = 10_000;
```

Comment:
Maybe making the delay shorter relative to the overall mining slot length would be better, in case it's a transient failure